core/jazz_services/swagger/swagger.json

```diff
@@ -542,6 +542,332 @@
         }
       }
     },
+    "/services/search": {
+      "options": {
+        "description": "CORS Support",
+        "consumes": [
```

Comment:
See POST method for /services/search which is not needed. 
![image](https://user-images.githubusercontent.com/1162453/68052330-a4226900-fca6-11e9-9c65-94660a3a32d9.png)

Reply:
Removed POST